test/expression_manipulation_test.jl

```diff
@@ -14,14 +14,14 @@ let
     GenX.fill_with_const!(small_const_expr, 6.0)
     @test small_const_expr == AffExpr.([6.0 6.0; 6.0 6.0; 6.0 6.0])
 
-    # Test createemptyexpression! with fill_with_const!
+    # Test create_empty_expression! with fill_with_const!
     large_dims = (2,10,20)
-    GenX.createemptyexpression!(EP, :large_expr, large_dims)
+    GenX.create_empty_expression!(EP, :large_expr, large_dims)
     @test all(EP[:large_expr] .== 0.0)
 
-    # Test createemptyexpression! with fill_with_const!
+    # Test create_empty_expression! with fill_with_const!
     large_dims_vector = collect(large_dims)
-    GenX.createemptyexpression!(EP, :large_const_expr, large_dims)
+    GenX.create_empty_expression!(EP, :large_const_expr, large_dims)
     GenX.fill_with_const!(EP[:large_const_expr], 6.0)
     @test all(EP[:large_const_expr] .== 6.0)
 
```

Comment:
Please make sure this PR is rebased w/ on the latest `develop` which incorporates modifications to this file